apps/settings/elements/improve_browser_os.html

```diff
@@ -33,6 +33,13 @@ <h2 data-l10n-id="performanceAndUsageData">Performance And Usage Data</h2>
             <span data-l10n-id="sharePerformanceAndUsageData">Submit Data</span>
           </label>
         </li>
+        <li>
+          <a id="openAppStats" class="menu-item">
+            <span data-l10n-id="openAppStats">
+              Open App Statistics
```

Comment:
I'll remove it for the new string, but would you mind filing a follow up for the others? I want to keep the impact of this patch low because it's coming in late